### ISSUE TITLE
parsing of "size_t const &"

### ISSUE THREAD

**Issue by syljua** [bytedeco/javacpp#173]
Parsing these prototypes;
```c++
size_t readData1(void *buffer, const size_t &count);
size_t readData2(void *buffer, size_t const &count);
size_t readData3(void *buffer, const size_t count);
size_t readData4(void *buffer, size_t const count);
```
Gives these results;
```java
public native @Cast("size_t") long readData1(Pointer buffer, @Cast("const size_t") long count);
public native @Cast("size_t") long readData2(Pointer buffer, @Cast("size_t const*") @ByRef SizeTPointer count);
public native @Cast("size_t") long readData3(Pointer buffer, @Cast("const size_t") long count);
public native @Cast("size_t") long readData4(Pointer buffer, @Cast("size_t const") long count);
```
Somethings wrong with the result for readData2, I think.

I might also be related to the effect of parsing this;
```c++
size_t readData7(void *buffer, int const &count);
```
That will give 3 different results (where I do not think I'm happy with none of them)
```java
public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef IntPointer count);
public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef IntBuffer count);
public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef int[] count);
```
Where this;
```c++
size_t readData8(void *buffer, const int &count); size_t readData8(void *buffer, const int &count);
```
is correctly done as;
```java
public native @Cast("size_t") long readData8(Pointer buffer, int count);
```
Thanks!

**Comment by syljua** [comment]
`public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef IntPointer count);
    public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef IntBuffer count);
    public native @Cast("size_t") long readData7(Pointer buffer, @Const({false, true}) @ByRef int[] count);
`

**Comment by saudet** [comment]
This should be fixed with the commit above. Let me know if you still have issues! Thanks

**Comment by saudet** [comment]
Fix released in version 1.3.3. Thanks for reporting!
